app/uk/gov/hmrc/exports/controllers/SubmissionController.scala

```diff
@@ -41,7 +42,25 @@ class SubmissionController @Inject()(
       authorizedWithEnrolment[SubmissionResponse](_ => processRequest)
     }
 
-  def processRequest()(implicit request: Request[SubmissionResponse], hc: HeaderCarrier): Future[Result] = {
+  def saveMovementSubmission(): Action[MovementResponse] =
+    Action.async(parse.json[MovementResponse](MovementResponse.format)) { implicit request =>
+      authorizedWithEnrolment[MovementResponse](_ => processSave)
+    }
+
+  private def processSave()(implicit request: Request[MovementResponse], hc: HeaderCarrier): Future[Result] = {
+    val body = request.body
+    movementsRepository.save(MovementSubmissions(body.eori, body.conversationId, body.ducr,body.mucr,body.movementType)).map(res =>
```

Comment:
Please remember splitting arguments with space. This part:`body.ducr,body.mucr,body.movementType)` is difficult to read, you have a lot of dots and commas